receiver/kafkametricsreceiver/metadata.yaml

```diff
@@ -0,0 +1,76 @@
+name: kafkametricsreceiver
+
+labels:
+  topic:
+    description: The ID (integer) of a topic
+  partition:
+    description: The number (integer) of the partition
+  group:
+    description: The ID (string) of a consumer group
+
+metrics:
+  kafka.brokers:
+    description: Number of brokers in the cluster.
+    unit: brokers
```

Comment:
Should this be `"{brokers}"` (looks like some of them are already that format but not all).

Reply:
I think the unit is not following our conventions https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/README.md#instrument-units so probably this should be "1" no unit?

Reply:
yeah my initial intention was leave unit-less, 
I went with the curly-braces as the `hostmetricsreceiver` uses a similar unit pattern, i.e for [{connections} in metadata.yaml](https://github.com/open-telemetry/opentelemetry-collector/blob/6e7827880cc4fd101e6783a9837c30bd2e7e717d/receiver/hostmetricsreceiver/metadata.yaml#L289)

Reply:
Actually, reviewing the units again, `{messages}` is not an accurate unit for an offset. It's a dimensionless measurement that kafka uses. Offset will use `1`.